packages/webapp/src/pages/Integrations/providerConfigKey/Show.tsx

```diff
@@ -0,0 +1,139 @@
+import { useParams, Link, Routes, Route, useLocation } from 'react-router-dom';
+import { LeftNavBarItems } from '../../../components/LeftNavBar';
+import DashboardLayout from '../../../layout/DashboardLayout';
+import Button from '../../../components/ui/button/Button';
+import { BookOpenIcon } from '@heroicons/react/24/outline';
+import IntegrationLogo from '../../../components/ui/IntegrationLogo';
+import { useStore } from '../../../store';
+import { PlusIcon } from '@radix-ui/react-icons';
+import { useGetIntegration } from '../../../hooks/useIntegration';
+import { Skeleton } from '../../../components/ui/Skeleton';
+import { Info } from '../../../components/Info';
+import PageNotFound from '../../PageNotFound';
+import { useEffect, useRef, useState } from 'react';
+import { EndpointsShow } from './Endpoints/Show';
+import { SettingsShow } from './Settings/Show';
+
+export const ShowIntegration: React.FC = () => {
+    const { providerConfigKey } = useParams();
+    const location = useLocation();
+    const ref = useRef<HTMLDivElement>(null);
+
+    const env = useStore((state) => state.env);
+
+    const { data, loading, error } = useGetIntegration(env, providerConfigKey!);
+    const [tab, setTab] = useState<string>('');
+
+    useEffect(() => {
+        if (location.pathname.match(/\/settings/)) {
```

Comment:
Think we should keep the same logic that is currently in place that if `isHosted` that the `Settings` tab loads by default instead of `Endpoints`: https://github.com/NangoHQ/nango/blob/master/packages/webapp/src/pages/Integration/Show.tsx#L57

Reply:
good point 👌🏻